docs/introduction.rst

```diff
@@ -4,8 +4,11 @@ Introduction
 
 We illustrate with an example how the OGGM workflow is applied to the
 Tasman Glacier in New Zealand (see figure below).
-Here we describe shortly the purpose of each processing step, while more
-details are provided in the other sections:
+Here we describe shortly the purpose of each processing steps, while more
```

Comment:
```suggestion
Here we describe shortly the purpose of each processing step, while more
```

I think singular is correct here?

Reply:
I think both are correct. However, it makes probably more sense to link the singular for step with the singular of purpose ...